pkg/api/datasources.go

```diff
@@ -385,10 +386,22 @@ func (hs *HTTPServer) CheckDatasourceHealth(c *models.ReqContext) {
 		return
 	}
 
+	var jsonDetails map[string]interface{}
 	payload := map[string]interface{}{
 		"status":      resp.Status.String(),
 		"message":     resp.Message,
-		"jsonDetails": resp.JSONDetails,
+		"jsonDetails": jsonDetails,
```

Comment:
could be renamed `details` but that would be a breaking change for the frontend.

Reply:
Go ahead with that  -- fro tend is ignoring it for now

Reply:
What timezone are you in? >.>